app/styles/_components.less

```diff
@@ -35,12 +35,9 @@
     margin-top: 15px;
   }
   .pod-template {
-    // so that .word-break() on children works
-    [flex] {
-      min-width: 0;
```

Comment:
wouldn't we need to keep `min-width: 0`?

Reply:
Yep.  It moved to line 53.